cmd/configmap-server/main.go

```diff
@@ -96,26 +103,34 @@ func runCmdFunc(cmd *cobra.Command, args []string) error {
 
 	configMapPopulator := sqlite.NewSQLLoaderForConfigMap(sqlLoader, *configMap)
 	if err := configMapPopulator.Populate(); err != nil {
-		return err
+		err = fmt.Errorf("error loading manifests from configmap: %s", err)
+		if !permissive {
+			logger.WithError(err).Fatal("permissive mode disabled")
+		}
+		logger.WithError(err).Warn("permissive mode enabled")
 	}
 
-	store, err := sqlite.NewSQLLiteQuerier(dbName)
+	var store registry.Query
+	store, err = sqlite.NewSQLLiteQuerier(dbName)
 	if err != nil {
-		logger.Fatalf("failed to load db: %v", err)
+		logger.WithError(err).Warnf("failed to load db")
+	}
+	if store == nil {
+		store = registry.NewEmptyQuerier()
 	}
 
 	// sanity check that the db is available
 	tables, err := store.ListTables(context.TODO())
 	if err != nil {
-		logger.Fatalf("couldn't list tables in db, incorrect config: %v", err)
+		logger.WithError(err).Warnf("couldn't list tables in db")
```

Comment:
Is there a chance of success at this point?

Reply:
At this point, the store is effectively empty. I added this to keep in line with our theme of being permissive where we can. I figured an empty store should be non-fatal. Do you think it makes sense for the `permissive` option to gate this branch point too?

Reply:
I was going back and forth on it.